scripts/compare-states/compare-states.go

```diff
@@ -167,7 +167,7 @@ func main() {
 	ctl.Usage = "compare-states RPC_A RPC_B"
 	ctl.Action = cliMain
 	ctl.Flags = []cli.Flag{
-		cli.BoolFlag{
+		&cli.BoolFlag{
 			Name:  "ignore-height, g",
 			Usage: "ignore height difference",
 		},
```

Comment:
Please, consider migrating from V1 name format to V2 name format *everywhere*. Check that *all* flags were migrated from `Name: "long, short"` to `Name: "long", Aliases:...`.